README.md

```diff
@@ -1,7 +1,7 @@
 # Fitness-Accountability
 
 - Name: Have this alphabetic
-- Why
+- Why: Why do you need Fitness Accountability
 - Weekly: What is your schedule
 - Goal
```

Comment:
```suggestion
- Goal: Please include a goal that is specific, measurable, and time-bound.
```

Reply:
This sounds like an instruction rather than a description - an alternative might be:
```
Goal: What is your specific, measurable, and time-bound goal
```